_sharp_basic_3/_sharp_basic_3/Program.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+
+class Homework_2
+{
+    static void Main()
+    {
+        //Task1
+        {
+            int enterNumber = Convert.ToInt32(Console.ReadLine());
+
+            if (enterNumber % 2 == 0 && enterNumber % 5 == 0)
+            {
+                Console.WriteLine("tutti-frutti");
+            }
+
+            else if (enterNumber % 5 == 0)
+            {
+                Console.WriteLine("frutti");
+            }
+
+            else if (enterNumber % 2 == 0)
+
+            {
+                Console.WriteLine("tutti");
+            }
+
+            Console.ReadLine();
+        }
+
+        //Task2
+        {

```

Comment:
It would be better to add additional verification if the secondNumber is greater than firstNumber.